boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi

```diff
@@ -103,32 +103,40 @@
 
 		boot_partition: partition@0 {
 			label = "mcuboot";
-			reg = <0x00000000 0x00020000>;
+			reg = <0x00000000 0x00008000>;
 			read-only;
 		};
-		slot0_partition: partition@20000 {
+		slot0_partition: partition@8000 {
 			label = "image-0";
-			reg = <0x00020000 0x00020000>;
+			reg = <0x00008000 0x00038000>;
 		};
+#ifndef CONFIG_TFM_BL2_FALSE
+		/* Leave 0x400 for MCUboot header */
+		slot1_partition: partition@40400 {
+			label = "image-1";
+			reg = <0x00040400 0x0001fc00>;
+		};
+#else
 		slot1_partition: partition@40000 {
 			label = "image-1";
-			reg = <0x00040000 0x00018000>;
+			reg = <0x00040000 0x00010000>;
 		};
-		slot2_partition: partition@58000 {
+#endif
```

Comment:
You can't use Kconfig in device tree: Device tree is processed before Kconfig.

Reply:
@erwango you are right, it does not work, I need to somehow offset the FLASH_LOAD_OFFSET variable with 0x400, mcuboot's header size.

I find that hard-coding the offset as a bad solution, there are too many independent places where address offsets are set (zephyr's dts, tfm's flash_layout.h) one more Kconfig is of no help.

Reply:
@erwango ROM_START_OFFSET was added in Kconfig.tfm that adds 0x400 as MCUboot's header, will remove the incorrect configuration.